_posts/0000-01-02-LukeW44.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
**Your about as useful as a one-legged man in an arse kicking contest - Mr. Bean**